src/backend/codegen/CMakeLists.txt

```diff
@@ -113,7 +127,7 @@ include_directories(${CLANG_INCLUDE_DIRS})
 # Core codegen library.
 add_library(gpcodegen SHARED
             utils/clang_compiler.cc
-            utils/code_generator.cc
+            utils/codegen_utils.cc
             ${codegen_tmpfile_sources})
```

Comment:
This came from Balerion - I believe its' used for debugging purposes. I'm fully sure.